pylint_odoo/checkers/modules_odoo.py

```diff
@@ -810,3 +825,49 @@ def _check_xml_attribute_translatable(self):
         if self.msg_args:
             return False
         return True
+
+    def _check_xml_deprecated_tree_attribute(self):
+        """The tree-view declaration is using a deprecated attribute.
+            Example <tree string="Partners"></tree>
+        """
+        checks = [
+            {
+                'attr': 'colors',
+                'skip_versions': {'4.2', '5.0', '6.0', '6.1', '7.0', '8.0'},
+                'xpath': './/tree[@colors]',
+            },
+            {
+                'attr': 'fonts',
+                'skip_versions': {'4.2', '5.0', '6.0', '6.1', '7.0', '8.0'},
+                'xpath': './/tree[@fonts]',
+            },
+            {
+                'attr': 'string',
+                'skip_versions': {'4.2', '5.0', '6.0', '6.1', '7.0'},
+                'xpath': './/tree[@string]',
+            },
+        ]
+        valid_versions = set(
+            self.linter._all_options['valid_odoo_versions'].config
+            .valid_odoo_versions)
+
+        def check_is_applicable(check):
+            return (check['attr'] in self.config.deprecated_tree_attributes and
+                    bool(valid_versions - check['skip_versions']))
```

Comment:
What about use the `__manifest__` version value for the module?

Reply:
Do you know if there is some helper to get the version from `__manifest__`?

Also, what if the version is invalid, eg. does not contain Odoo version (`1.0.0` or even `8.0.0` where `8` is the major version of the module)?